packages/react-ds/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+
+import classNames from 'classnames/bind'
+
+import { IInput } from '@venice/core/models/input.model'
+import styles from '@venice/styles/components/Input.module.scss'
+
+const Input: FC<IInput> = ({
```

Comment:
Use `React.FunctionComponent` instead of `FC`. It's the same thing but with a more expressive name.

Reply:
I think we should decouple the `container` div from here and create another component for that.

Also we could use the `Input` component internally and expose components with the `type` prop already defined, such as `type = 'text'` via export.

i.e.: `<InputText />`, `<InputPassword />`